### PR TITLE
revert CUMULUS-3540

### PR DESCRIPTION
**Pull request by etcart**
This reverts CUMULUS-3540, turning back on previously failing interfaces with cmr in integration tests
this reverts:
gh: https://github.com/nasa/cumulus/pull/3544
jira: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3540

* correctly un-stub functions in integration-tests/cmr.js
  * getOnlineResources
  * conceptExists
  * waitForConceptExistsOutcome
* turn back on postToCmr sub-tests 

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by Jkovarik** [comment]
Note: this is reverting https://github.com/nasa/cumulus/pull/3543

**Comment by Jkovarik** [comment]
The revert being referred to here is https://github.com/nasa/cumulus/pull/3543

**Comment by Jkovarik** [comment]
Ticket is https://bugs.earthdata.nasa.gov/browse/CUMULUS-3540 , linked ticket is  https://bugs.earthdata.nasa.gov/browse/CUMULUS-3555

File: example/cumulus-tf/lambdas.tf [hunk 1]
```diff
@@ -148,6 +148,7 @@ resource "aws_lambda_function" "ftpPopulateTestLambda" {
   role             = module.cumulus.lambda_processing_role_arn
   runtime          = "nodejs16.x"
   timeout          = 150
+  memory_size      = 512
```
Comment (Jkovarik) [inline example/cumulus-tf/lambdas.tf:151]: This mod should be included in the CL.   I realize it's not user facing, but it's project facing nonetheless.

**Comment by etcart** [comment]
other way around, this ticket is CUMULUS-3555, reverting to CUMULUS-3540, I'll make that clearer in the description

File: package.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
   },
   "devDependencies": {
     "@ava/typescript": "^1.1.1",
-    "@babel/core": "^7.9.6",
+    "@babel/core": "^7.23.2",
```
Comment (Jkovarik) [inline package.json:78]: What was the reason for this update?   CL entry with details please.

File: tasks/orca-copy-to-archive-adapter/package.json [hunk 3]
```diff
@@ -42,7 +42,7 @@
     "@cumulus/common": "18.1.0",
     "@cumulus/cumulus-message-adapter-js": "2.0.5",
     "@cumulus/logger": "18.1.0",
-    "lodash": "^4.17.15"
+    "lodash": "^4.17.19"
```
Comment (Jkovarik) [inline tasks/orca-copy-to-archive-adapter/package.json:45]: What was the reason for uppinning this?   CL entry/etc, please.

**Review by Jkovarik — CHANGES_REQUESTED** [review]
Looks good, there are a couple of changes that are *additional* to a revert of #3544, and we should have a CL like what we had in that PR noting the change back/remove the entry if it wasn't included in a release.

**Review by Jkovarik — APPROVED** [review]
Looks good to me - thanks for the updates

**Pull request merged**